bundles/net.tourbook/src/net/tourbook/ui/views/tourBook/TVITourBookItem.java

```diff
@@ -146,6 +149,7 @@
 
    long         colAltitudeUp;
    long         colAltitudeDown;
+   float        colAvgAltitudeChange;
```

Comment:
My suggestion for the name is `colAltitude_AvgChange` similar to `colTemperature_...`